mis_builder/models/aep.py

```diff
@@ -28,6 +28,13 @@ class AccountingExpressionProcessor(object):
         * accounts is a list of accounts, possibly containing % wildcards
         * an optional domain on move lines allowing filters on eg analytic
           accounts or journal
+        * an optional parameter for rate calculation. this parameter will
+          override the default value set in the report instance.
+          possible values are
+          empty or now: current date exchange ratenow,
+          start: start date of period (date_from)
+          end: end date of period (date_to)
+          daily: daily rate of move line date.
```

Comment:
I understand the need to have different currency rate conversion methods in different kpis of the same report. But I'm not too comfortable with having that in the formula itself. From some quick reading I just did, I'd think it's more a property of the GL account. What do you think?